### PR TITLE
join: modify log level

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

modify log level

### Check List

Tests

- Unit test

### Release note

- No release note

### CODE REVIEW

File: server/join/join.go [hunk 1]
```diff
@@ -214,7 +214,7 @@ func PrepareJoinCluster(cfg *config.Config) error {
 func isDataExist(d string) bool {
 	dir, err := os.Open(d)
 	if err != nil {
-		log.Error("failed to open directory", errs.ZapError(errs.ErrOSOpen, err))
+		log.Info("failed to open directory, maybe first start", errs.ZapError(errs.ErrOSOpen, err))
```
Comment (HunDunDM) [inline server/join/join.go:217]:
```suggestion
		log.Info("failed to open directory, maybe start for the first time", zap.Error(err))
```
Not an Error log, so no need to use `errs`

**Review by JmPotato — APPROVED** [review]

**Review by HunDunDM — APPROVED** [review]

**Comment by HunDunDM** [comment]
fix #3579

**Pull request merged**